vllm/engine/output_processor/stop_checker.py

```diff
@@ -85,33 +89,36 @@ def maybe_stop_sequence(
             return
 
     @staticmethod
-    def _check_stop_strings(seq: Sequence, new_char_count: int,
-                            sampling_params: SamplingParams) -> Optional[str]:
+    def check_stop_strings(
```

Comment:
@njhill - can you add unit tests for this?

Reply:
It's covered by https://github.com/vllm-project/vllm/blob/main/tests/engine/test_stop_strings.py, but we could add some more isolated tests too